contracts/Registrar/PartnerRegistrar.sol

```diff
@@ -27,7 +27,7 @@ contract PartnerRegistrar is
 
     NodeOwner private _nodeOwner;
     IERC677 private _rif;
-    IPartnerManager private _partnerManager;
+    IPartnerManager public partnerManager;
```

Comment:
the only observation I would make is that the get partnermanager property becomes part of the API. We need to also define it in the INTERFACE. In order to ensure future implementation remain compatible.

Reply:
so... Maybe it becomes a function? getPartnerManager?

Reply:
Yes, then the property can remain `private`. That's better.

Reply:
We should expose the API only via the interface.